### PR TITLE
ui: logo and top menu fixes

### PR DESCRIPTION
**Pull request by kpsherva**

### CODE REVIEW

File: ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js [hunk 1]
```diff
@@ -51,11 +71,12 @@ export default class ILSMenu extends Component {
         content={'Sign in'}
       />
     ) : (
-      this.renderRightDropDown(userMenuText)
+      <>{this.renderRightDropDown(userMenuText)}</>
     );
   };
 
   render() {
+    console.log(this.props.user);
```
Comment (KonstantinaStoikou) [inline ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js]: Is there a reason for this `console.log`?

**Review by KonstantinaStoikou — APPROVED** [review]

File: ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js [hunk 2]
```diff
@@ -18,27 +25,40 @@ export default class ILSMenu extends Component {
   };
 
   renderRightDropDown = userMenuText => {
+    const dropdownEntries = (
+      <Dropdown.Menu>
+        <Dropdown.Item as={Link} to={FrontSiteRoutes.patronProfile}>
+          Your Profile
```
Comment (ntarocco) [inline ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js]:
```suggestion
          Your loans
```
We will have a conflict anyway

File: ui/src/authentication/pages/ConfirmEmail/ConfirmEmail.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ const Confirmed = ({ isConfirmed }) => {
         <Link className="alternative" to={FrontSiteRoutes.home}>
           home page
         </Link>{' '}
-        to browse out catalogue
+        to browse our catalogue
```
Comment (ntarocco) [inline ui/src/authentication/pages/ConfirmEmail/ConfirmEmail.js]:
```suggestion
        to browse the library catalogue
```

File: ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js [hunk 4]
```diff
@@ -68,7 +89,9 @@ export default class ILSMenu extends Component {
           >
             <Container>
               <Menu.Item header>
-                <Link to="/">ILS</Link>
+                <Link to="/">
+                  <Image src={logo} size="tiny" centered alt="ILS" />
```
Comment (ntarocco) [inline ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js]:
```suggestion
                  <Image src={logo} size="tiny" centered alt="Logo" />
```

File: ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js [hunk 5]
```diff
@@ -84,7 +107,9 @@ export default class ILSMenu extends Component {
           <Menu borderless inverted fixed="top" className="mobile-header-menu">
             <Container>
               <Menu.Item header>
-                <Link to="/">ILS</Link>
+                <Link to="/">
+                  <Image src={logo} size="tiny" centered alt="ILS" />
```
Comment (ntarocco) [inline ui/src/pages/frontsite/components/ILSMenu/ILSMenu.js]:
```suggestion
                  <Image src={logo} size="tiny" centered alt="Logo" />
```

File: ui/src/semantic-ui/site/globals/site.variables [hunk 6]
```diff
@@ -21,7 +21,7 @@
 ********************************/
 
 
-@headerMenuHeight         : 69px;
+@headerMenuHeight         : 73px;
```
Comment (ntarocco) [inline ui/src/semantic-ui/site/globals/site.variables:24]: :+1:

**Review by ntarocco — COMMENTED** [review]
Thanks for fixing this. A couple of comments:
* any chance we can display the [name](https://github.com/inveniosoftware/invenio-app-ils/blob/master/invenio_app_ils/records/api.py#L392) instead of Hello, <email>?
* can the user understand that the `Hello, <email>` is a clickable dropdown menu? 
  * I would go for simply the name with the arrow down: `Mario Rossi ▾`
  * another solution would be to simply display the links one after the other and remove the dropdown entirely:
     `LOGO` ----------------------------------------- `Your loans`   `Backoffice`   `Sign out`

**Comment by kpsherva** [comment]
> Any chance to fix this horrible banner on the top?
> 
> localhost_3000_login_sessionExpired

this is a bigger issue, I would attack it in a separate PR
basically it is common for the backoffice and frontsite and ir requires some refactoring to split it.

**Review by ntarocco — APPROVED** [review]

**Pull request merged**